j2k/src/org/jetbrains/kotlin/j2k/Converter.kt

```diff
@@ -424,6 +425,24 @@ class Converter private constructor(
         }
     }
 
+
+    private fun specialAnnotationPropertyCases(field: PsiField): Annotations {
+        val javaSerializableInterface = PsiType.getTypeByName("java.io.Serializable", project, GlobalSearchScope.allScope(project)).resolve()
```

Comment:
It's a minor issue, but the correct way is to use `field.getResolveScope()` instead of `allScope` here. Otherwise you may get incorrect results if the project uses multiple JDKs in different modules.